prow/config.yaml

```diff
@@ -1,10 +1,7 @@
 ---
 plank:
-  # TODO(fejta): switch to spyglass
-  #ob_url_template: 'https://prow.istio.io/view/gcs/istio-prow/{{if eq .Spec.Type "presubmit"}}pull/{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}{{else}}.{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}/'
-  #ob_url_prefix: https://prow.istio.io/view/gcs
-  job_url_template: 'https://gubernator.k8s.io/build/istio-prow/{{if eq .Spec.Type "presubmit"}}pull/{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}{{else}}.{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}/'
-  job_url_prefix: https://gubernator.k8s.io/build/
+  job_url_template: 'https://prow.istio.io/view/gcs/istio-prow/{{if eq .Spec.Type "presubmit"}}pr-logs/pull/{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}{{else}}logs/{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}'
+  job_url_prefix: https://prow.istio.io/view/gcs
```

Comment:
Unfortunately I think this needs a trailing slash to be correct for the pubsub reporter.
https://github.com/kubernetes/test-infra/issues/13211#issuecomment-507412018

This should really work as is though. The fact that the pubsub reporter makes an assumption about this is a bug IMO.

Reply:
Done